src/parameter.c

```diff
@@ -405,6 +412,7 @@ int parse_arguments(struct ntttcp_test *test, int argc, char **argv)
 		{"show-nic-packets", required_argument, NULL, LO_SHOW_NIC_PACKETS},
 		{"show-dev-interrupts", required_argument, NULL, LO_SHOW_DEV_INTERRUPTS},
 		{"fq-rate-limit", required_argument, NULL, LO_FQ_RATE_LIMIT},
+		{"no-delay", no_argument, NULL, LO_TCP_NO_DELAY},
```

Comment:
I don't see the need for continuing to support the non-TCP_NODELAY case, and vote for unconditionally always set TCP_NODELAY on all sockets.

We had some discussion on https://github.com/sagargadher-microsoft/ntttcp-for-linux-tcp-nodelay/pulls/1 but it has been removed since.